sig-bug-triage-process.md

```diff
@@ -0,0 +1,54 @@
+## SIG Bug Triage Process
+
+SIGs should be triaging the issues with their matching label during every SIG
+meeting. You can view
+[a list of recently filed issues](https://github.com/spinnaker/spinnaker/labels/sig%2Fplatform)
+on GitHub to and click through the ones filed since your last meeting. (That
+link shows issues for the Platform SIG, but you can easily edit the filter from
+there.)
+
+Each issue should have one of three outcomes:
+
+1.  **Assigned to a user or added to an actively-maintained GitHub project.** An
+    actively-maintained project has a team a people who are using it to plan
+    future work. If you assign it to a project, you should also add the
+    `no-lifecycle` tag so that Spinnakerbot doesn't automatically close it.
+
+1.  **Marked with the label "contributions welcome".** This is an indicator that
+    we don't know anyone who will be working on the issue, but we would be
+    interested in discussing it further if a contributor could be found. Please
+    also add a message to the bug describing the situation:
+
+    > Thanks for filing the issue. Unfortunately, no one is available to work on
+    > this at the moment, but we welcome contributions from our community.
+    > Before starting work on it, we'd strongly encourage you to discuss your
+    > plans here or on the SIG's [slack channel](https://join.spinnaker.io/).
+
+    Also add the tag `beginner friendly` if it applies. Add `no-lifecycle` if
```

Comment:
I think that @maggieneterval made spinnakerbot ignore `beginner friendly` tasks so that we don't auto-close these and can keep a pool of good first tasks.

Reply:
I didn't really mean for those two sentences to go together. I reordered them and hopefully it makes more sense now.